docs/build_from_source.md

```diff
@@ -0,0 +1,100 @@
+# Build BladeDISC from Source in a Docker container
+
+This document introduced how to build BladeDISC from source,
+to easy the software installation and configuration, we provide a
+[Dockerfile](/docker/dev/Dockerfile) that contains requirements software
+to build and test.
+
+## Prerequisite
+
+1. Git for checking out the source code.
+1. [Nvidia Docker](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html)
+to launch Docker container on Nvidia GPU host.
+
+## Checkout the Source
+
+``` bash
+git clone git@github.com:alibaba/BladeDISC.git
+cd BladeDISC && git submodule update --init --recursive
+```
+
+## Building BladeDISC for TensorFlow Wrapper
+
+1. step0: launch a development Docker container that runs a
```

Comment:
although we are IT engineers, i still feel it's weird to count from step0 ...  ^^